### PR TITLE
Point to new version 3.6

### PR DESCRIPTION
**Pull request by serbioboeronfido**
Added a new minor version 3.6.0
Point all urls to this new version in the library

### CODE REVIEW

**Review by DavidMealha — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## v2.8.0, 19 January 202
+
+- Updated to use API v3.6, for more details please see our [release notes](https://developers.onfido.com/release-notes#api-v35).
```
Comment (dvacca) [inline CHANGELOG.md:3]: README.md should be updated as welll

Comment (serbioboeronfido) [reply CHANGELOG.md]: Corrected.

**Comment by danielolivaresd** [comment]
Friendly reminder to update new endpoints introduced in https://github.com/onfido/onfido-ruby/pull/42 when you rebase

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+## v2.8.0, 24 January 2023
+
+- Updated to use API v3.6, for more details please see our [release notes](https://developers.onfido.com/release-notes#api-v36).
+- Add Motion support
```
Comment (DavidMealha) [inline CHANGELOG.md:4]: Motion was already released on 2.5.0

Comment (serbioboeronfido) [reply CHANGELOG.md]: Yes True. Removing it.

**Review by dvacca — APPROVED** [review]

**Comment by danielolivaresd** [comment]
@DavidMealha friendly reminder that this PR adds a new version but I think we're missing the actual (gem) release for 2.8.0, just in case you didn't notice.

**Comment by serbioboeronfido** [comment]
@danielolivaresd Released. Please check [rubygems](https://rubygems.org/gems/onfido)

**Pull request merged**